docs/alerting-recording.md

```diff
@@ -0,0 +1,63 @@
+# Alerting & Recording Rules
+
+### Alerting Rules
+
+Alerting rules are used to trigger alerts based on the violation of any condition(s). These alerts are fired to external services like slack, mails, etc by the alert manager (link). Alerting rules are written in a YAML file and paths of these files are mentioned in the Prometheus configuration respectively. It is important to note that the evaluation of these conditional rules are done on the Prometheus side.
```

Comment:
```suggestion
Alerting rules are used to trigger alerts based on the violation of any condition(s). These alerts are fired to external services like slack, mails, etc by the alert manager (link). Alerting rules are written in a YAML file and paths of these files are mentioned in the Prometheus configuration respectively. It is important to note that the evaluation of these conditional rules are done at the Prometheus side.
```

1. Does this sound better with `at`?
2. Please add the link to the alert manager.